code/SCINGE.m

```diff
@@ -12,31 +12,13 @@
 % Outputs:
 % ranked_edges = ranked list of gene interactions with corresponding SCINGE scores
 % gene_influence = ranked lists of regulators (genes) with corresponding SCINGE influence
-SCINGE_version = '0.2.0';
+SCINGE_version = '0.3.0';
 display(SCINGE_version);
 for rep = 1:num_replicates
     for ii = 1:length(param_list)
-        GLG_Instance(Data,'lambda',param_list{ii}.lambda,'dT',param_list{ii}.dT,'num_lags',param_list{ii}.num_lags,'kernel_width',param_list{ii}.kernel_width,'prob_zero_removal',param_list{ii}.prob_zero_removal,'replicate',rep,'ID',param_list{ii}.ID,'outdir',outdir,'family',param_list{ii}.family,'prob_remove_samples',param_list{ii}.prob_remove_samples,'date',param_list{ii}.date);
+        GLG_Instance(Data,'--lambda',param_list{ii}.lambda,'--dT',param_list{ii}.dT,'--num_lags',param_list{ii}.num_lags,'--kernel_width',param_list{ii}.kernel_width,'--prob_zero_removal',param_list{ii}.prob_zero_removal,'--replicate',rep,'--ID',param_list{ii}.ID,'--outdir',outdir,'--family',param_list{ii}.family,'--prob_remove_samples',param_list{ii}.prob_remove_samples,'--date',param_list{ii}.date);
```

Comment:
Should we switch to `-` instead of `_` for multi-word arguments?  I believe that is more standard for long arguments.